internal/mode/static/manager.go

```diff
@@ -530,7 +530,7 @@ func registerControllers(
 }
 
 // 10 min jitter is enough per telemetry destination recommendation
-// For the default period of 24 hours, jitter will be 10min /(24*60)min  = 0.0069
+// For the default period of 24 hours, jitter will be 10min /(24*60)min  = 0.0069.
```

Comment:
I'm not convinced that having periods at the end of comments inside functions is a big deal. I'm okay with them being on function or type definition comments, but is there any way to require those but not these?

Or maybe I'm in the minority here.

Reply:
It's only checking top level declaration comments

Reply:
Oh, duh. I'm not sure why I saw this one differently.

Reply:
> I'm okay with them being on function or type definition comments, but is there any way to require those but not these?

👍